src/compas/datastructures/network/planarity.py

```diff
@@ -27,18 +27,9 @@
     'network_is_planar',
     'network_is_planar_embedding',
     'network_embed_in_plane',
-
-    'network_embed_in_plane_proxy',
 ]
 
 
-def network_embed_in_plane_proxy(data, fixed=None, straightline=True):
```

Comment:
Removing this function would make it a breaking change, unless this is duplicated and also provided by some other file? Otherwise I would suggest to add it back.

Reply:
sure i can put it back, but this `proxy` version of functions is just a left over from old code...